rpcserver.go

```diff
@@ -1907,6 +1907,21 @@ func (r *rpcServer) parseOpenChannelReq(in *lnrpc.OpenChannelRequest,
 
 	localFundingAmt := btcutil.Amount(in.LocalFundingAmount)
 	remoteInitialBalance := btcutil.Amount(in.PushSat)
+	minHtlcIn := lnwire.MilliSatoshi(in.MinHtlcMsat)
```

Comment:
I don't think these need to be moved up. Otherwise we don't see at one glance that we don't need these values in the sanity checks.